java-client/src/test/java/org/opensearch/client/opensearch/integTest/AbstractCatClientIT.java

```diff
@@ -209,11 +215,48 @@ public void testCatSegments() throws Exception {
         assertNull(segmentsRecord.ip());
         assertNull(segmentsRecord.prirep());
     }
+    
+    @Test
+    public void testCatPitSegments() throws Exception {
+        InfoResponse info = javaClient().info();
+        String version = info.version().number();
+        if (version.contains("SNAPSHOT")) {
+                version = version.split("-")[0];
+        }
+        if (Version.fromString(version).onOrAfter(Version.fromString("2.4.0"))) {
```

Comment:
You could probably use `assumeThat` or `assumeTrue` instead to ignore the test for older OS versions:

```
assumeTrue("The PIT is supported in OpenSearch 2.4.0 and later", Version.fromString(version).onOrAfter(Version.fromString("2.4.0"));
```

Otherwise it gives an impression that test passes but it has to be ignored otherwise

Reply:
Done!